api/placer.py

```diff
@@ -93,6 +93,8 @@ def check(self):
         self.saved = []
 
     def process_file_field(self, field, info):
+        for x in ('type', 'instrument', 'measurements', 'tags', 'metadata'):
```

Comment:
Ah yes I did this when I thought things other than this whitelist might be in metadata, this can be removed and `info.update(self.metadata)` can replace it.